Implement the caesar cipher algorithm and Longest Common subsequence in python

manthan0227 opened the pull request:
# Caesar Cipher
Added Caesar cipher algorithm using python language.

☑Hacktoberfest
☑Contributor

close #179
 ## Describe the changes you have made
Added Caesar cipher algorithm using python language.

# Longest Common Subsequence
Added Longest Common Subsequence using recursion in python.

☑Hacktoberfest
☑Contributor

close #172 
 ## Describe the changes you have made
Longest Common subsequence added in dynamic programming directory in python language.

prathimacode-hub commented:
1 issue = 1 PR. Don't add multiple issues in single PR. Kindly correct it. @manthan0227

manthan0227 commented:
### Caesar Cipher
Added Caesar cipher algorithm using python language.

☑Hacktoberfest
☑Contributor

close #179

#### Describe the changes you have made
Added Caesar cipher algorithm using python language.

prathimacode-hub commented:
No PR yet from you. @manthan0227
